README.md

```diff
@@ -160,6 +161,58 @@ export class AppComponent {
 ></highcharts-chart>
 ```
 
+### Angular Elements and useHTML
+
+First, install angular elements:
+
+```cli
+npm install @angular/elements --save
+```
+
+Include in main.ts file your element tag inside allowedTags and [element properties](https://angular.io/guide/elements#mapping) inside allowedAttributes:
+
+```ts
+if (Highcharts && Highcharts.AST) {
+  Highcharts.AST.allowedTags.push('translation-element');
+  Highcharts.AST.allowedAttributes.push('translation-key');
+}
+```
+
+Define your element in the constructor of the component that will use it:
+
+```ts
+private defineTranslationElement() {
+  if (isNil(customElements.get('translation-element'))) {
+    const translationElement = createCustomElement(TranslationComponent, {
+      injector: this.injector,
+    });
+    customElements.define('translation-element', translationElement);
+  }
+}
+```
+
+Then, create the element, set properties and use it in the chart:
+
+```ts
+const translationEl: NgElement & WithProperties<TranslationComponent> =
+      document.createElement(translationElementTag) as any;
```

Comment:
Sorry for nitpicking but can we please avoid this `any` casting?

Reply:
Sure, thank you @karolkolodziej. Casting removed from [master](https://github.com/jonancastelo/highcharts-angular) branch on [jonancastelo/highcharts-angular](https://github.com/jonancastelo/highcharts-angular).